app/routes/report.js

```diff
@@ -121,6 +81,7 @@ module.exports = [{
             return {
               frn: hold.frn,
               scheme: hold.holdCategorySchemeName,
+              marketingYear: hold.marketingYear,
```

Comment:
If we change this to `marketingYear: hold.marketingYear ?? 'All'` will it be safe to deploy this without needing the corresponding `ffc-pay-processing` change?